### ISSUE TITLE
invalid orderId when invalidating an order on the taker side

### ISSUE THREAD

**Issue by offerm** [ExchangeUnion/xud#480]
Should be check once #468 is merged
>9/14/2018, 2:35:37 PM [ORDERBOOK] warn: incoming order invalidation: invalid orderId (4adf4b31-b812-11e8-943e-2fe2d882e096)

Besides the direct issue, consider also why this is a warning and not an error.

```
9/14/2018, 2:35:37 PM [SWAPS] info: ResolveHash starting with hash: 9d7778979c48d0d57f62efa6c98df1d4065e481d1ee84137015e163373674b18
9/14/2018, 2:35:37 PM [SWAPS] debug: Executing taker code
9/14/2018, 2:35:37 PM [SWAPS] debug: Amount received for preImage b5aad427d945dfba4f0f4bb0e4f002cf7981a82fb6018124168372401bdfd9b9
9/14/2018, 2:35:37 PM [P2P] verbose: canceled order from 0235584e2ac9a34a13ce6f6dc9fb8aa1e7918d7701f0b4b69dc002887e6b3fbba3: {"orderId":"4adf4b31-b812-11e8-943e-2fe2d882e096","pairId":"LTC/BTC"}
9/14/2018, 2:35:37 PM [ORDERBOOK] warn: incoming order invalidation: invalid orderId (4adf4b31-b812-11e8-943e-2fe2d882e096)
9/14/2018, 2:35:37 PM [SWAPS] debug: Swap completed. preimage = b5aad427d945dfba4f0f4bb0e4f002cf7981a82fb6018124168372401bdfd9b9
9/14/2018, 2:35:37 PM [SWAPS] debug: Sending swap complete to peer: {"r_hash":"9d7778979c48d0d57f62efa6c98df1d4065e481d1ee84137015e163373674b18"}
```

**Comment by sangaman** [comment]
Duplicate of #478. I think `warn` is appropriate given that nothing actually failed and no functionality is compromised, it's just an unexpected message from a peer.
